modin/engines/ray/cudf_on_ray/frame/data.py

```diff
@@ -16,11 +16,11 @@
 from .partition import cuDFOnRayFramePartition
 from .partition_manager import cuDFOnRayFrameManager
 
-from modin.engines.base.frame.data import BasePandasFrame
+from modin.engines.ray.frame.data import PandasOnRayFrame
 from modin.error_message import ErrorMessage
 
 
-class cuDFOnRayFrame(BasePandasFrame):
+class cuDFOnRayFrame(PandasOnRayFrame):
```

Comment:
Why are we inherent from de PandasOnRayFrame instead of de BasePandasFrame? If I remember correctly, the PandasOnRayFrame only has one method and the rest gets inherent from the BasePandasFrame. In my opinion, we should inherent from the Base and then override those methods on the BasePandasFrame similar to the PandasOnRayFrame

Reply:
So the biggest reason is that pandasOnRayFrame exposes the `combine_dtypes` api which would be useful for cudfOnRayFrame. But other than that, you're right in that there's no point.

Reply:
I see! Well if that's the case I think this is fine!